guacamole-common-js/src/main/webapp/modules/Tunnel.js

```diff
@@ -823,6 +749,8 @@ Guacamole.WebSocketTunnel = function(tunnelURL) {
      */
     var tunnel = this;
 
+    var parser = null;
```

Comment:
Guessing this should get some documentation?

Reply:
Oops - yes, definitely.

Reply:
Fixed via rebase.